conf/env.sh.example

```diff
@@ -1,4 +1,5 @@
 #! /usr/bin/env bash
+# Variable UNO_HOME already defined, either by user or uno location in load-env.sh
```

Comment:
Nice comment.  Could say `Variable UNO_HOME is already defined in load-env.sh to root directory of this repo.`